site/public/js/forum.js

```diff
@@ -1085,6 +1085,8 @@ function loadThreadHandler(){
                     return;
                 }
 
+                $(obj).find(".thread_box").css("background", "white");
```

Comment:
Could you do `$(obj).find('.thread_box').removeClass('new_thread');` here instead?